web/server/api/endpoints/models.py

```diff
@@ -18,15 +24,43 @@ def get_models(
         models.Model(
             name=model.name,
             path=str(model._path.relative_to(context.path)),
-            description=model.description,
-            owner=model.owner,
-            dialect=model.dialect,
+            dialect=model.dialect if model.dialect else "Default",
             columns=[
                 models.Column(
                     name=name, type=str(data_type), description=model.column_descriptions.get(name)
                 )
                 for name, data_type in model.columns_to_types.items()
             ],
+            details=models.ModelDetails(
+                owner=model.owner,
+                kind=model.kind.name,
+                batch_size=model.batch_size,
+                cron=model.cron,
+                stamp=model.stamp,
+                start=model.start,
+                retention=model.retention,
+                storage_format=model.storage_format,
+                time_column=f"{model.time_column.column} | {model.time_column.format}"
```

Comment:
These inline arguments are a little difficult to read. Can you please move them into variables, i.e. time_column and type?

Reply:
sorry , did not get it , move where?

Reply:
Something like
```python
response = []
for model in context.models.values():
    time_column = f"{model.time_column.column} | {model.time_column.format}" if model.time_column else None
    details = models.ModelDetails(
        ...,
        time_column=time_column,
    )
    response.append(
        models.Model(
            name=model.name,
            ...,
            details=details,
    )
return response
```